src/ensembl/src/content/app/species-selector/state/speciesSelectorEpics.ts

```diff
@@ -47,14 +47,18 @@ export const fetchSpeciesSearchResultsEpic: Epic<Action, Action, RootState> = (
       // but forget the previous query every time user clears search results
       (action1, action2) =>
         action1.type === action2.type &&
-        (action1 as PayloadAction<
-          'species_selector/species_search_request',
-          string
-        >).payload ===
-          (action2 as PayloadAction<
+        (
+          action1 as PayloadAction<
             'species_selector/species_search_request',
             string
-          >).payload
+          >
+        ).payload ===
+          (
+            action2 as PayloadAction<
+              'species_selector/species_search_request',
+              string
+            >
+          ).payload
```

Comment:
Is this just prettier? If so, could you please revert this change to avoid confusion?

Reply:
Probably.  Will do it.